apps/ideaspace/src/components/modules/EditIdea/EditIdea.js

```diff
@@ -0,0 +1,192 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { useRouter } from 'next/router';
+import { useUserDataContext } from '@devlaunchers/components/context/UserDataContext';
+
+import SignInSection from '../../common/SignInSection/SignInSection';
+import BackButton from '../../common/BackButton/BackButton';
+import IdeaForm from '../../common/IdeaForm/IdeaForm';
+import useConfirm from '../../common/DialogBox/DialogBox';
+import getNotice from '../../common/DialogBox/NoticeBox';
+import * as Yup from 'yup';
+import { atoms } from '@devlaunchers/components/src/components';
+
+import {
+  HeadWapper,
+} from '../SubmissionForm/StyledSubmissionForm';
+
+function SubmissionForm() {
+  let { userData, setUserData, isAuthenticated } = useUserDataContext();
+  if (process.env.NEXT_PUBLIC_NAME == 'DEVELOPMENT') {
+    isAuthenticated = true;
+
+    React.useEffect(() => {
+      setUserData({ ...userData, id: 2 });
+    }, []);
+  }
+
+  const router = useRouter();
+  const { ideaId } = router.query;
+  const [sending, setSending] = React.useState(false);
+  const [unsavedChanges, setunsavedChanges] = useState(false);
+  const [Dialog, confirmLeave] = useConfirm(
+    'You have unsaved changes',
+    'Are you sure you want to discard the changes and leave?',
+  );
+  const [urrl, setUrrl] = useState('');
+
+  const [Notice, confirmNotice] = getNotice(
+    'Idea updated successfully',
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
```

Comment:
@alicefate Is this "Lorem ipsum" text going to be displayed to users? I think we should remove it unless there is a specific reason to keep it.

Reply:
It will show to users. The content is from UXD design. I have no idea what it means.

Reply:
Removed it.